.github/workflows/build.yml

```diff
@@ -502,3 +502,269 @@ jobs:
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0 # so that `git describe` works.
+      - name: Set environment variables
+        run: |
+          $ENV:CESIUM_UNREAL_VERSION=$(git describe)
+          $ENV:BUILD_CESIUM_UNREAL_PACKAGE_NAME="CesiumForUnreal-52-android-${ENV:CESIUM_UNREAL_VERSION}"
+          # Make these available to subsequent steps
+          echo "CESIUM_UNREAL_VERSION=${ENV:CESIUM_UNREAL_VERSION}" >> $ENV:GITHUB_ENV
+          echo "BUILD_CESIUM_UNREAL_PACKAGE_NAME=${ENV:BUILD_CESIUM_UNREAL_PACKAGE_NAME}" >> $ENV:GITHUB_ENV
+      - name: Install nasm
+        uses: ilammy/setup-nasm@v1.4.0
+      - name: Build cesium-native
+        run: |
+          cd extern
+          cmake -B build-android -S . -G Ninja -DCMAKE_TOOLCHAIN_FILE="unreal-android-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Release
+          cmake --build build-android --config Release --target install -j8
+      - name: Build plugin
+        run: |
+          ((Get-Content -path CesiumForUnreal.uplugin -Raw) -replace '"EngineVersion": "5.0.0"','"EngineVersion": "5.2.0"') | Set-Content -Path CesiumForUnreal.uplugin
+          cd "C:/Program Files/Epic Games/UE_5.2/Engine/Build/BatchFiles"
+          ./RunUAT.bat BuildPlugin -Plugin="$ENV:GITHUB_WORKSPACE/CesiumForUnreal.uplugin" -Package="$ENV:GITHUB_WORKSPACE/packages/CesiumForUnreal" -CreateSubFolder -TargetPlatforms=Android -NoHostPlatform
+      - name: Publish plugin package artifact
+        if: ${{ success() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ env.BUILD_CESIUM_UNREAL_PACKAGE_NAME}}.zip
+          path: packages
+  Linux52:
+    runs-on: ["self-hosted","linux","x64","unreal-52"]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          fetch-depth: 0 # so that `git describe` works.
+      - name: Set environment variables
+        run: |
+          export CESIUM_UNREAL_VERSION=$(git describe)
+          export BUILD_CESIUM_UNREAL_PACKAGE_NAME="CesiumForUnreal-52-linux-${CESIUM_UNREAL_VERSION}"
+          # Make these available to subsequent steps
+          echo "CESIUM_UNREAL_VERSION=${CESIUM_UNREAL_VERSION}" >> "$GITHUB_ENV"
+          echo "BUILD_CESIUM_UNREAL_PACKAGE_NAME=${BUILD_CESIUM_UNREAL_PACKAGE_NAME}" >> "$GITHUB_ENV"
+      - name: Install nasm
+        uses: ilammy/setup-nasm@v1.4.0
+      - name: Build cesium-native
+        run: |
+          export UNREAL_ENGINE_DIR="/opt/ue52"
+          export LINUX_MULTIARCH_ROOT="/opt/ue52/Engine/Extras/ThirdPartyNotUE/SDKs/HostLinux/Linux_x64/v21_clang-15.0.1-centos7/"
+          export UNREAL_ENGINE_COMPILER_DIR="${LINUX_MULTIARCH_ROOT}x86_64-unknown-linux-gnu"
+          export UNREAL_ENGINE_LIBCXX_DIR="${UNREAL_ENGINE_DIR}/Engine/Source/ThirdParty/Unix/LibCxx"
+          # TODO: Move this to the image instead
```

Comment:
Does this TODO need to be addressed before this PR is merged?

Reply:
No, not urgent. It would just make the build a tiny bit faster, at the cost of lots of time building a new image. I'll do it next time I have some other reason to change the image.